### PR TITLE
Fix flaws in HTTP/

### PR DESCRIPTION
**Pull request by teoli2003**
I fixed about half of the flaws under the HTTP/ hierarchy.

(We plan to improve it in the near future, cleaning it a bit is a good start)

### CODE REVIEW

File: files/en-us/web/http/headers/large-allocation/index.html [hunk 1]
```diff
@@ -105,11 +105,9 @@ <h2 id="Troubleshooting_errors">Troubleshooting errors</h2>
   </dd>
   <dt>A <code>Large-Allocation</code> header was ignored due to the document not being
     loaded out of process.</dt>
-  <dd>Firefox has moved to a <a
-      href="/en-US/docs/Mozilla/Firefox/Multiprocess_Firefox">multiprocess
-      architecture</a>, and this architecture is required in order to support the
-    <code>Large-Allocation</code> header. Some <a
-      href="/en-US/docs/Mozilla/Add-ons/SDK">legacy Addons</a> can prevent Firefox from
+  <dd>Firefox has moved to a multiprocess
```
Comment (wbamberg) [inline files/en-us/web/http/headers/large-allocation/index.html:108]: I reckon we can delete this whole `dt/dd` now, this hasn't been relevant for years.

**Review by wbamberg — APPROVED** [review]
This looks fine.

I had one suggestion to delete the thing that talks about legacy Firefox add-ons. If you want to do that, I'm happy for you to do it then merge. Otherwise I'm happy for you to merge this as it is :).

File: files/en-us/web/http/headers/large-allocation/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/http/headers/large-allocation/index.html]: Done

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Pull request merged**